scripts/tools/InitializeQueue.php

```diff
@@ -66,9 +67,17 @@ class InitializeQueue extends InstallAction
 {
     use ServiceLocatorAwareTrait;
 
-    const BROKER_MEMORY = 'memory';
-    const BROKER_RDS = 'rds';
-    const BROKER_SQS = 'sqs';
+    public const BROKER_MEMORY = 'memory';
+    public const BROKER_RDS = 'rds';
+    public const BROKER_NEW_SQL = 'newsql';
```

Comment:
- [ ] Please validate that the `newSql` broker requires a persistence too in the method `checkParams`.

Reply:
done